src/pages/BrowseSchemes.svelte

```diff
@@ -129,6 +149,21 @@
     }
 
     schemes = Object.values(byScheme);
+
+    let set1 = new Set();
+    let set2 = new Set();
+    for (let x of schemes) {
+      if (x.authority_or_region) {
+        set1.add(x.authority_or_region);
+      }
+      if (x.funding_programme) {
+        set2.add(x.funding_programme);
+      }
+    }
+    authorities = Array.from(set1.entries());
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/entries
Option value and display text may differ, so `Select` takes `[string, string][]`, and `entries` conveniently does this for us